source/simulator/solver.cc

```diff
@@ -841,22 +875,25 @@ namespace aspect
 
         solver_control_cheap.enable_history_data();
         solver_control_expensive.enable_history_data();
+        internal::InverseWeightedMassMatrix<TrilinosWrappers::PreconditionBase> inverse_weighted_mass_matrix(system_matrix.block(1,1),*Mp_preconditioner,parameters.linear_solver_S_block_tolerance);
```

Comment:
please break this line into several lines, each constructor parameter on a new line.